lib/manager/gomod/update.ts

```diff
@@ -77,6 +77,8 @@ export function updateDependency(
         upgrade.newMajor > 1 &&
         !newLine.includes(`/v${upgrade.newMajor}`)
       ) {
+        // If package has no version, pin to latest one.
+        newLine = newLine.replace(depName, depName + '/v' + upgrade.newMajor);
```

Comment:
@mangas Is there any documentation for the new suffix handling for go modules? Does this work for all go modules?

Reply:
Renovate already handles the /v suffix, in this case just doesn't handle updates from no suffix to the latest suffix which means the run fails and PR doesn't get generated

Reply:
https://github.com/golang/go/wiki/Modules#what-are-some-implications-of-tagging-my-project-with-major-version-v0-v1-or-making-breaking-changes-with-v2

Reply:
This link has all the info about releases and semvers